.github/workflows/sonarcloud.yml

```diff
@@ -11,7 +11,8 @@ on:
       - '**.md'
       - '**.adoc'
       - '*.txt'
-      - '.ci/**'      
+      - '.ci/**'
+      - '**/optaplanner-examples/**/swingui/**/*'
```

Comment:
This configuration just defines when the CI for a regular SonarCloud check is skipped. Usually, it runs after a PR has been merged to update the `main` branch analysis (in contrast to the PR check analysis, which checks just the changes introduces in a PR when it's open).

Reply:
Yes, sorry I got it wrong